integration-test/1491-greek-road-shields.py

```diff
@@ -0,0 +1,408 @@
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'name': u'Εθνική Οδός 7 (Κόρινθος - Καλαμάτα)',
+                'name:en': u'National Highway 7 (Corinth - Kalamata)',
+                'name:fr': u'Route Nationale 7 (Corinthe - Kalamata)',
+                'network': u'GR:national',
+                'ref': u'ΕΟ7',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q487890',
+            }, ways=[25114417]),
+            dsl.relation(2, {
+                'int_name': u'Ethniki Odos 82 (Sparti - Kalamata - Pylos)',
+                'name': u'Εθνική Οδός 82 (Σπάρτη - Καλαμάτα - Πύλος)',
+                'name:en': u'National Highway 82 (Sparta - Kalamata - Pylos)',
+                'name:fr': u'Route Nationale 82 (Sparte - Kalamata - Pylos)',
+                'network': u'GR:national',
+                'ref': u'ΕΟ82',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[25114417]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 25114417,
+                'network': u'GR:national',
+                'shield_text': u'7',
+                'all_networks': ['GR:national', 'GR:national'],
+                'all_shield_texts': ['7', '82'],
+            })
+
+    def test_8a_grnational(self):
+        # checking that we keep the trailing alpha
+        import dsl
+
+        z, x, y = (16, 36730, 25214)
+
+        self.generate_fixtures(
+            dsl.is_in('GR', z, x, y),
+            # https://www.openstreetmap.org/way/19832532
+            dsl.way(19832532, dsl.tile_diagonal(z, x, y), {
+                'highway': u'trunk',
+                'lanes': u'2',
+                'lit': u'yes',
+                'maxspeed': u'50',
+                'name': u'Νέα Εθνική Οδός Πατρών - Αθηνών',
+                'oneway': u'yes',
+                'ref': u'ΕΟ8α',
+                'smoothness': u'excellent',
+                'source': u'openstreetmap.org',
+                'source:ref': u'OKXE',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'int_name': u'Korinthos - Patra',
+                'name': u'Εθνική Οδός 8α (Κόρινθος - Πάτρα)',
+                'name:en': u'National Highway 8α (Korinthos - Patra)',
+                'name:fr': u'Route nationale 8α (Corinthe - Patras)',
+                'network': u'GR:national',
+                'ref': u'ΕΟ8α',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[19832532]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 19832532,
+                'network': u'GR:national',
+                'shield_text': u'8α',
```

Comment:
Just curious: what does the `α` signify?

Reply:
As far as I can tell, [the EO8a was built as a grade-separated road paralleling route of the EO8](https://el.wikipedia.org/wiki/%CE%95%CE%B8%CE%BD%CE%B9%CE%BA%CE%AE_%CE%9F%CE%B4%CF%8C%CF%82_8%CE%B1_(%CE%95%CE%BB%CE%BB%CE%AC%CE%B4%CE%B1)), but at some point most of it was renamed to A8 and there are only a few stretches keeping the name EO8a (presumably which didn't make sense as part of the A-network).